index.html

```diff
@@ -681,18 +681,18 @@
 </p>
                       <!--script(src=dir+'/controller.js')-->
                       <script>app.controller('BsdateCtrl', function($scope) {
-	$scope.user = {
```

Comment:
Tabs were changed to spaces here.If this is unacceptable, please let me know.  As for now, i'm just going to leave it since it doesn't affect functionality, and tabs are the devil. :)